syntax = "proto3";


package proto_banmap;


message BanEntry {
    uint32 version = 1;
    int64 create_time = 2;
    int64 ban_until = 3;
    uint32 ban_reason = 4;
}

message BanMap {
    map<string, BanEntry> map = 1;
}
